pkg/fanal/types/const.go

```diff
@@ -23,6 +23,7 @@ const (
 	Pnpm       = "pnpm"
 	Jar        = "jar"
 	Pom        = "pom"
+	GradleLock = "gradle-lock"
```

Comment:
I think Gradle is enough.

```suggestion
	Gradle = "gradle"
```

Reply:
Done